recipes/quest/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "quest" %}
+{% set version = "3.0-a1" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erdc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  skip: True
```

Comment:
This is skipping all the builds, please remove of select which platform or python you want to skip

Reply:
How do you skip python 2.7 ?

Reply:
If this is a pure python project you can just remove the ``skip: True`` and in ``host`` and also in ``run`` you put something like ``python >3``

Reply:
@marcelotrevisani thank you for your help and advice. Oh you even pulled my PR and updated it for me. Thank you so much sir!